pom.xml

```diff
@@ -117,8 +117,10 @@
         <kxml2.servicemix.version>2.3.0_3</kxml2.servicemix.version>
         <!-- double-check downstream projects before changing jackson version -->
         <fasterxml.jackson.version>2.7.5</fasterxml.jackson.version>
-        <cxf.version>3.1.4</cxf.version>
-        <httpclient.version>4.4.1</httpclient.version>
+        <cxf.version>3.1.10</cxf.version>
+        <httpcomponents.httpclient.version>4.5.2</httpcomponents.httpclient.version>
+        <httpcomponents.httpcore.version>4.4.4</httpcomponents.httpcore.version>
+        <httpclient.version>4.4.1</httpclient.version> <!-- kept for compatibility in 0.11.0-SNAPSHOT, remove after -->
```

Comment:
should this be `4.5.2` for good measure?  (we don't have any way to mark it is deprecated do we?)